src/context/WalletProvider/DemoWallet/config.ts

```diff
@@ -3,7 +3,7 @@ import { FoxIcon } from 'components/Icons/FoxIcon'
 import type { SupportedWalletInfo } from 'context/WalletProvider/config'
 
 export const DemoConfig: Omit<SupportedWalletInfo, 'routes'> = {
-  adapter: NativeAdapter,
+  adapter: [NativeAdapter],
```

Comment:
if we're going to make this an array, please pluralize. you can do `F2`/rename symbol in vscode or most ideas to propagate this throughout.

Reply:
hmm, you mind doing this for me? i'm a webstormer

Reply:
Shift + F6, fellow WebStormer.

Reply:
@BitHighlander is the guidance from @0xApotheosis helpful for you to do this?